benchmark/src/test/java/ai/timefold/solver/benchmark/impl/aggregator/BenchmarkAggregatorTest.java

```diff
@@ -0,0 +1,258 @@
+package ai.timefold.solver.benchmark.impl.aggregator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import ai.timefold.solver.benchmark.config.PlannerBenchmarkConfig;
+import ai.timefold.solver.benchmark.config.SolverBenchmarkConfig;
+import ai.timefold.solver.benchmark.config.report.BenchmarkReportConfig;
+import ai.timefold.solver.core.config.solver.SolverConfig;
+import ai.timefold.solver.core.config.solver.termination.TerminationConfig;
+import ai.timefold.solver.core.testdomain.TestdataEasyScoreCalculator;
+import ai.timefold.solver.core.testdomain.TestdataEntity;
+import ai.timefold.solver.core.testdomain.TestdataSolution;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class BenchmarkAggregatorTest {
+
+    private BenchmarkAggregator benchmarkAggregator;
+
+    @BeforeEach
+    void setUp() {
+        benchmarkAggregator = new BenchmarkAggregator();
+    }
```

Comment:
Arguably unnecessary. This will do the same thing:

    private final BenchmarkAggregator benchmarkAggregator = new BenchmarkAggregator();